fontir/src/ir.rs

```diff
@@ -54,8 +58,13 @@ impl StaticMetadata {
     ) -> Result<StaticMetadata, VariationModelError> {
         let axis_names = axes.iter().map(|a| a.name.clone()).collect();
         let variation_model = VariationModel::new(glyph_locations, axis_names)?;
+        // https://github.com/googlefonts/ufo2ft/blob/fca66fe3ea1ea88ffb36f8264b21ce042d3afd05/Lib/ufo2ft/fontInfoData.py#L38-L45
+        let ascender = (0.8 * units_per_em as f32).into();
+        let descender = (0.2 * units_per_em as f32).into();
```

Comment:
note that those ufo2ft's fallback ascender/descender are ot-rounded, not left as floats. It doesn't look like you are rounding them here.

Reply:
well you _are_ rounding them before writing them to hhea. I wonder if you should round them already in the IR as well.

Reply:
I think for IR, we could type those fields as i16, I doubt anybody wants these to ever be floats (like upem)

Reply:
I made it negative and rounded in hhea. In the light of a new day I think the negative part is in the wrong place but the rounding makes sense to me to defer.